go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/stackitcloud/stackit-cli
 go 1.21
 
 require (
+	github.com/depp/bytesize v1.1.0
```

Comment:
This is not used, right?

Reply:
You can run `go mod tidy` in the repo root to clear unused dependencies from the `go.mod` file